static/elements/chromedash-upcoming.js

```diff
@@ -0,0 +1,95 @@
+import {LitElement, html} from 'lit-element';
+import '@polymer/iron-icon';
+import style from '../css/elements/chromedash-upcoming.css';
+
+const TEMPLATE_CONTENT = {
+  stable: {
+    channelLabel: 'Stable',
+    h1Class: '',
+    downloadUrl: 'https://www.google.com/chrome/browser/desktop/index.html',
+    downloadTitle: 'Download Chrome stable',
+    dateText: 'was',
+    featureHeader: 'Features in this release',
+  },
+  beta: {
+    channelLabel: 'Next up',
+    h1Class: 'chrome_version--beta',
+    downloadUrl: 'https://www.google.com/chrome/browser/beta.html',
+    downloadTitle: 'Download Chrome Beta',
+    dateText: 'between',
+    featureHeader: 'Features planned in this release',
+  },
+  dev: {
+    channelLabel: 'Dev',
+    h1Class: 'chrome_version--dev',
+    downloadUrl: 'https://www.google.com/chrome/browser/canary.html',
+    downloadTitle: 'Download Chrome Canary',
+    dateText: 'coming',
+    featureHeader: 'Features planned in this release',
+  },
+};
+const REMOVED_STATUS = ['Removed'];
+const DEPRECATED_STATUS = ['Deprecated', 'No longer pursuing'];
+const SHOW_DATES = true;
+
+class ChromedashUpcoming extends LitElement {
+  static styles = style;
+
+  static get properties() {
+    return {
+      // Assigned in upcoming-page.js,
+      channels: {attribute: false},
+      showBlink: {attribute: false},
+      signedIn: {type: Boolean},
+      loginUrl: {type: String},
+      starredFeatures: {type: Object},
+    };
+  }
+
+  constructor() {
+    super();
+  }
+
+  // Handles the Star-Toggle event fired by any one of the child components
+  handleStarToggle(e) {
+    const newStarredFeatures = new Set(this.starredFeatures);
+    window.csClient.setStar(e.detail.feature, e.detail.doStar)
+      .then(() => {
+        if (e.detail.doStar) {
+          newStarredFeatures.add(e.detail.feature);
+        } else {
+          newStarredFeatures.delete(e.detail.feature);
+        }
+        this.starredFeatures = newStarredFeatures;
+      })
+      .catch(() => {
+        alert('Unable to star the Feature. Please Try Again.');
+      });
+  }
+
+
+  render() {
+    if (!this.channels) {
+      return html``;
+    }
+
+    return html`
+      ${['stable', 'beta', 'dev'].map((type) => html`
+        <chromedash-upcoming-milestone-card
+          channel=${JSON.stringify(this.channels[type])}
```

Comment:
Rather than convert objects to a string set attributes, I think you can set properties directly using a leading period.  For example, I think you could have
  `.channel=${this.channels[type]}`

Of course, what you have will work and it might be easier to debug, but it seems like it could create an opportunity for future defects if there is ever any value that cannot be serialized.

See: https://lit-html.polymer-project.org/guide/writing-templates#bind-to-properties

Reply:
Okay. Thanks for letting me know. 

Done. See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1432/commits/8f2bfe4d221537bac662172628161ddf06c5d14c) commit

Reply:
Why do some values still have single quotes around the expression?  Is that needed?

Reply:
Oops... I had removed them. But later, when I undo-ed some of the changes, I forgot to remove them again. Thanks for pointing it out. 

Done

Reply:
Looks good.